Assets/Scripts/PlayerController.cs

```diff
@@ -119,9 +113,16 @@ public void Pickup()
 
     public void Damage()
     {
-        health = health - 1;
-        Debug.Log(health);
-        
+        if (health <= 1)
+        {
+            InstaKill();
+        }
+
+        else
+        {
+            health = health - 1;
```

Comment:
This health deduction should happen first and then the health < 1 check should happen.